grails-app/services/au/org/emii/portal/BulkDownloadService.groovy

```diff
@@ -145,4 +145,14 @@ class BulkDownloadService {
         zipStream.write bytes, 0, bytes.length
         zipStream.closeEntry()
     }
+
+    public static void copy(final InputStream input, final OutputStream output, final int bufferSize) throws IOException
```

Comment:
@jkburges This is the only function we need from apache.commons.io. So I took it. But we can also pull apache.commons.io - either way works for me. the grails `IOUtils.copy` function closes both streams when it finishes, which is not good for us.